batchflow/models/torch/base.py

```diff
@@ -1828,6 +1887,11 @@ def load(self, file, make_infrastructure=False, mode='eval', pickle_module=dill,
                 self._loaded_from_onnx = True
                 self.disable_training = True
 
+            if "safetensors" in checkpoint:
+                from safetensors.torch import load_file
+                state_dict = load_file(checkpoint['path_safetensors'], device=device)
+                self.model.load_state_dict(state_dict)
```

Comment:
Did you test `load` method? As i understand, `self.model` is `None` before that line and that line will raise Exception